src/main/java/core/basesyntax/model/machine/Car.java

```diff
@@ -1,9 +1,29 @@
 package core.basesyntax.model.machine;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+
+@Entity
+@Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
 public class Car extends Machine {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
let's review Car, Truck and Machine classes (the last two are not changed at all)
`@Inheritance` and `@Id` should be present in the parent class